scripts/docker_run

```diff
@@ -3,11 +3,19 @@
 set -o errexit
 set -o xtrace
 
-docker build --tag=oak .
+USERNAME=$(id -un)
+
+mkdir -p $PWD/bazel-cache
+touch $PWD/bazel-cache/.placeholder
```

Comment:
We either need to `mkdir`, *or* add the placeholder file and check it in, not both (I think).

Reply:
Done